app/controllers/mou_signatures_controller.rb

```diff
@@ -1,9 +1,48 @@
 class MouSignaturesController < ApplicationController
-  after_action :verify_authorized
+  after_action :verify_authorized, only: %i[index]
```

Comment:
If none of the other actions need authorization then its ok to remove this before action.

Reply:
as in just delete it?

Reply:
I'm going to merge this as it is and tackle this in a follow-up PR if thats ok. I have another small issue I need to fix so I'll throw it in with that.